add(clickhouse): how to query a query log (#550)
aiven/aiven-docs: docs/products/clickhouse/howto/manage-databases-tables.md

engines to ensure high availability. Even if you select `MergeTree`
engine, we will automatically use the replicated variant on our side.

:::note

> A query log is a non-replicated table. Running a query again a query log requires

Reviewer: 1. again**ST**?
2. query log -> `system.query_log`?

> A non-replicated table, such as `system.query_log`, you can
> [query using `clusterAllReplicas`](/docs/products/clickhouse/howto/query-databases#query-a-non-replicated-table).

Reviewer 2: ```suggestion
To query non-replicated tables, such as `system.query_log`, use [`clusterAllReplicas`](/docs/products/clickhouse/howto/query-databases#query-a-non-replicated-table).
```
How about this, more direct?

> :::

Reviewer: I would say that for querying non replicated tables, (like query log) clusterAllReplicas might be used, and for shards - distributed table might be used.
